src/index.js

```diff
@@ -791,6 +832,31 @@ export default class Gantt {
         $bar.finaldx = 0;
       });
     });
+    $.on(this.$container, 'scroll', e => {
```

Comment:
sticky header is choppy probably because frames are skipped between renders, try doing this inside requestAnimationFrame